cinn/auto_schedule/measure/measure.h

```diff
@@ -41,10 +42,13 @@ struct MeasureInput {
 struct MeasureResult {
   // The time cost of execution in average of running
   // with a specific repeated times.
-  double execution_cost;  // unit: us
+  double execution_cost = 0.0;  // unit: us
```

Comment:
I didn't see the usage of `execution_cost` in this PR? Where do you plan to use it?

Reply:
It will be set in `ScheduleRunner::Run`, an implement can be found at `measure/simple_runner.cc`, which  has been completed in previous pr